Modified development document (EN) for Windows users (#7171)
open-telemetry/opentelemetry.io: content/en/docs/contributing/development.md

Builds are officially supported on Linux-based environments and macOS. Other
environments, such as [DevContainers](#devcontainers), are supported on a

> best-effort basis. For builds on Windows similar steps as of Linux can be
> followed on Windows Subsystem for Linux command line [WSL][windows-wsl].

Reviewer: ```suggestion
best-effort basis. For builds on Windows, you can follow steps similar to those for Linux using Windows Subsystem for Linux command line [WSL][windows-wsl].
```
